ui/plugin-admin/src/components/Plugin.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { DataGridRow, DataGridCell } from "juno-ui-components"
+import { useGlobalsActions, useShowDetailsFor } from "./StoreProvider"
+
+import { Icon } from "juno-ui-components"
+
+// renders a single plugin row
+const Plugin = (props) => {
+  const plugin = props.plugin
+  const { setShowDetailsFor } = useGlobalsActions()
+  const showDetailsFor = useShowDetailsFor()
+
+  const showDetails = () => {
+    showDetailsFor === plugin.id
+      ? setShowDetailsFor(null)
+      : setShowDetailsFor(plugin.id)
+  }
+
+  return (
+    <DataGridRow
+      key={plugin.id}
+      onClick={showDetails}
+      className={`cursor-pointer ${
+        showDetailsFor === plugin.id ? "active" : ""
+      } ${plugin?.disabled ? "text-theme-disabled" : ""} `}
+    >
+      <DataGridCell>
+        <p>{plugin.name}</p>
```

Comment:
wrapping the cell's content in a `<p>` tag is not necessary. The only reason you would want to wrap a cell's content in a wrapper element is if you have multiple children within the cell and you don't want the default vertical flexbox layout applied to it. See documentation for the `cellVerticalAlignment` prop of the `DataGrid` component.